lib/smart_proxy_openscap/store_arf_spool.rb

```diff
@@ -0,0 +1,20 @@
+require 'smart_proxy_openscap/store_arf_fs'
+module Proxy::OpenSCAP
+  class StoreArfSpool < StoreArfFS
+    def store(cn, id, date, data)
+      begin
+        target_dir = create_directory(Proxy::OpenSCAP::Plugin.settings.spooldir, cn, id, date)
+      rescue StandardError => e
+        raise Proxy::OpenSCAP::StoreSpoolError.new "Could not fulfill request: #{e.message}"
```

Comment:
`raise Proxy::OpenSCAP::StoreSpoolError, "Could not fulfill request: #{e.message}"`